src/operation/decision_model.jl

```diff
@@ -271,6 +277,33 @@ end
 
 function validate_time_series(model::DecisionModel{<:DefaultDecisionProblem})
     sys = get_system(model)
+    settings = get_settings(model)
+    available_resolutions = PSY.list_time_series_resolutions(sys)
+
+    if get_resolution(settings) == UNSET_RESOLUTION && length(available_resolutions) != 1
+        throw(
+            IS.ConflictingInputsError(
+                "Data contains multiple resolutions, the resolution keyword argument must be added to the Model. Time Series Resolutions: $(available_resolutions)",
+            ),
+        )
+    elseif get_resolution(settings) != UNSET_RESOLUTION && length(available_resolutions) > 1
+        if get_resolution(settings) ∉ available_resolutions
```

Comment:
Doesn't this check also need to happen when there is only one available resolution?

Reply:
This check is in case someone passes a Resolution to the model specification that isn't in the data

Reply:
What if the user passes a resolution that isn't in the data and there is only 1 available resolution?

Reply:
Error for ConflictingArguments

Reply:
I'll check, maybe the logic is wrong.